samples/snippets/src/main/java/com/example/mediatranslation/TranslateFromFile.java

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright 2020 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.mediatranslation;
+
+// [START media_translation_translate_from_file]
+
+import com.google.api.gax.rpc.BidiStream;
+import com.google.cloud.mediatranslation.v1beta1.SpeechTranslationServiceClient;
+import com.google.cloud.mediatranslation.v1beta1.StreamingTranslateSpeechConfig;
+import com.google.cloud.mediatranslation.v1beta1.StreamingTranslateSpeechRequest;
+import com.google.cloud.mediatranslation.v1beta1.StreamingTranslateSpeechResponse;
+import com.google.cloud.mediatranslation.v1beta1.StreamingTranslateSpeechResult;
+import com.google.cloud.mediatranslation.v1beta1.TranslateSpeechConfig;
+import com.google.protobuf.ByteString;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+public class TranslateFromFile {
+
+  public static void translateFromFile() throws IOException {
+    // TODO(developer): Replace these variables before running the sample.
+    String filePath = "path/to/audio.raw";
+    translateFromFile(filePath);
+  }
+
+  public static void translateFromFile(String filePath) throws IOException {
+    try (SpeechTranslationServiceClient speechTranslationServiceClient =
+        SpeechTranslationServiceClient.create()) {
+      Path path = Paths.get(filePath);
+      byte[] content = Files.readAllBytes(path);
+      ByteString audioContent = ByteString.copyFrom(content);
+
+      TranslateSpeechConfig translateSpeechConfig =
+          TranslateSpeechConfig.newBuilder()
+              .setAudioEncoding("linear16")
```

Comment:
This seems wrong, shouldn't it be an enum?
Like this line?
https://github.com/GoogleCloudPlatform/java-docs-samples/blob/master/speech/cloud-client/src/main/java/com/example/speech/Recognize.java#L139

Reply:
Currently I cannot find it from ` com.google.cloud.mediatranslation.v1beta1`. I will reach out prod team about this

Reply:
Got response back from them: `no, it is string type, we use enum at first, but API reviewer suggested to use string as we don't need to release API again if we have new types`

Should we leave as it is or Should I import other library to use their enum?

Reply:
Can't use a different library that won't work. 
Um can you plus one me to that email thread. I'd like to know if this is gonna be the new norm going forward or something got slipped and we should fix now.